build/gulp/plugins/gulp-example-source.ts

```diff
@@ -4,7 +4,7 @@ import * as prettier from 'prettier'
 import * as through from 'through2'
 import * as Vinyl from 'vinyl'
 
-import * as prettierConfig from '../../../.prettierrc.json'
+const prettierConfig = require('../../../.prettierrc.json')
```

Comment:
could you, please, suggest why this change was necessary?

Reply:
Now we have a specific `tsconfig.json` for `react-proptypes` that doesn't include `/build` directory, so it will fail with: 
```
../../build/gulp/plugins/gulp-example-source.ts(7,33): error TS2732: Cannot find modu
le '../../../.prettierrc.json'. Consider using '--resolveJsonModule' to import module
 with '.json' extension
```

Did I miss something?